### DISCUSSION TITLE
Scaling algorithm/smoothing tweaks

### DISCUSSION THREAD

**Discussion by MrJaunty** [rbreu/beeref#56] [Questions & Issues] [answered]
Beeref offers great functionality and, most importantly, speed. The scaling algorithm is the only thing preventing me from using beeref exclusively. Would it be possible to either tweak the algorithm used to scale when zooming in/out, or allow for different scaling algos to be selected from? Currently, jagged scaling is noticeable on photos, however line-art is where the issue really becomes visible. 

Viewing zoomed-out lineart in beeref is effectively unusable, as the way the images are scaled break up most lines and can render fine lines entirely invisible. I've tried to attach an image in this post that illustrates the difference in behaviour between the same images when viewed in beeref and in pureref at similar levels of zoom. 

![image](https://user-images.githubusercontent.com/3712532/158320875-da48a617-95f1-4002-88c2-ff028d7a8e2b.png)

This is genuinely the single issue in beeref that holds it back from being the default tool for me.

**Comment by KannebTo** [comment, marked as answer]
Hey @MrJaunty, I addressed this problem in PR #60. Feel free to test it.
